front/src/modules/trainschedule/styles/ManageTrainSchedule/_itinerary.scss

```diff
@@ -58,10 +58,24 @@
       }
 
       &-name {
-        max-width: 47%;
+        width: 45%;
```

Comment:
I feel like the behavior is better with 40%

Reply:
that's a UI decision, we should be careful with this kind of suggestions based on personal preferences